src/locales/en/translation.json

```diff
@@ -152,6 +152,8 @@
     "value": "{{value, number}}"
   },
   "errors": {
+    "canNotLoadData": "Can not load data",
+    "canNotReachDataSource": "We were unable to reach the data source to request this data. This is probably a temporary (network?) problem. Please try again later.",
```

Comment:
> "(network?)"

should we avoid the word "network" and use connection/connectivity?

cc @donouwens

Reply:
Friendly reminder: this is waiting for input from @donouwens